js/verifySignup.js

```diff
@@ -0,0 +1,125 @@
+const email = document.querySelector("#email");
+const failLoginBlank = document.querySelector(".fail-login-blank");
+const failLoginEmail = document.querySelector(".fail-login-email");
+const nickName = document.querySelector("#nickname");
+const failNicknameBlank = document.querySelector(".fail-nickname-blank");
+const password = document.querySelector("#password");
+const failPasswordBlank = document.querySelector(".fail-password-blank");
+const failPasswordLength = document.querySelector(".fail-password-length");
+const passwordCheck = document.querySelector("#passwordCheck");
+const failPasswordCheck = document.querySelector(".fail-password-check");
+const signupButton = document.querySelector(".signup-button");
+const signupLink = document.querySelector("#signup-success-link");
+
+signupButton.onclick = function() {
+  if(checkBlankType(email.value)) {
+    failLoginBlank.classList.remove("hide");
+    failLoginEmail.classList.add("hide");
+    email.classList.add("fail-layout");
+  } else if(checkBlankType(nickName.value)) {
+    failNicknameBlank.classList.remove("hide");
+    nickName.classList.add("fail-layout");
+  } else if(checkBlankType(password.value)) {
+    failPasswordBlank.classList.remove("hide");
+    failPasswordLength.classList.add("hide");
+    password.classList.add("fail-layout");
+  }
+};
+
+email.onfocus = function() {
+  failLoginBlank.classList.add("hide");
+  failLoginEmail.classList.add("hide");
+  email.classList.remove("fail-layout");
+};
+
+email.onblur = function() {
+  if(checkBlankType(email.value)) {
+    failLoginBlank.classList.remove("hide");
+    failLoginEmail.classList.add("hide");
+    email.classList.add("fail-layout");
+  } else if(!checkEmailType(email.value)) {
+    failLoginEmail.classList.remove("hide");
+    failLoginBlank.classList.add("hide");
+    email.classList.add("fail-layout");
+  }
+  checkAll()
+};
```

Comment:
`const email = document.querySelector("#email");` email 변수를 사용되는 곳에 가까이 위치시키면 가독성이 더 좋아집니다. 
```suggestion
const email = document.querySelector("#email");
email.onfocus = function() {
  failLoginBlank.classList.add("hide");
  failLoginEmail.classList.add("hide");
  email.classList.remove("fail-layout");
};

email.onblur = function() {
  if(checkBlankType(email.value)) {
    failLoginBlank.classList.remove("hide");
    failLoginEmail.classList.add("hide");
    email.classList.add("fail-layout");
  } else if(!checkEmailType(email.value)) {
    failLoginEmail.classList.remove("hide");
    failLoginBlank.classList.add("hide");
    email.classList.add("fail-layout");
  }
  checkAll()
};
```